app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="sale_off">Giảm giá</string>
+    <string name="email">Email: </string>
+    <string name="chatwork_id">Mã Chatwork: </string>
+    <string name="join_date">Ngày tham gia: </string>
+    <string name="status">Trạng thái: </string>
+    <string name="description">Mô tả: </string>
+    <string name="new_password">Mật khẩu mới</string>
+    <string name="confirm_new_password">Xác nhận mật khẩu mới</string>
+    <string name="current_password">Mật khẩu hiện tại</string>
+    <string name="submit">Xác nhận</string>
+    <string name="money_unit">\u0020VNĐ</string>
+    <string name="login_error">Lỗi đăng nhập !</string>
+    <string name="login_success">Đăng nhập thành công !</string>
+    <string name="wrong_email_or_password">Không thể tìm thấy người dùng!</string>
+    <string name="message">Tin nhắn</string>
+    <string name="ok">OK</string>
+    <string name="cannot_load_the_data">Không thể tải dữ liệu</string>
+    <string name="email_is_Empty">Email không được để trống !</string>
+    <string name="password_is_Empty">Mật khẩu không được để trống!</string>
+    <string name="you_need_to_choose_a_domain_to_continue">Bạn cần chọn miền để tiếp tục!</string>
+    <string name="none">Không</string>
+    <string name="search">Tìm kiếm...</string>
+    <string name="product">Sản phẩm</string>
+    <string name="shop">Cửa hàng</string>
+    <string name="please_click_back_again_to_exit">Làm ơn bấm Back lần nữa để thoát!</string>
+    <string name="confirm_new_password_does_not_match">Xác nhận mật khẩu mới không trùng khớp</string>
+    <string name="current_password_is_empty">Mật khẩu hiện tại để trống</string>
+    <string name="update_profile_success">Cập nhật thông tin cá nhân thành công</string>
+    <string name="nothing_change">Không có gì thay đổi</string>
+    <string name="change_password">Thay đổi mật khẩu</string>
+    <string name="change_domain">Thay đổi miền</string>
+    <string name="shop_name">Tên cửa hàng: </string>
+    <string name="product_quantity">Số lượng sản phẩm: </string>
+    <string name="owner">Chủ cửa hàng: </string>
+    <string name="rating">Xếp hạng: </string>
+    <string name="comments">Bình luận</string>
+    <string name="products_shop">Sản phẩm trong shop</string>
+    <string name="see_all">Xem tất cả</string>
+    <string name="title_shop">Cửa hàng :\u0020 </string>
+    <string name="shop_information">Thông tin cửa hàng</string>
+    <string name="shop_owner_information">Thông tin chủ cửa hàng</string>
+    <string name="time">Thời gian:</string>
+    <string name="pending">Chờ</string>
+    <string name="accepted">Chấp nhận</string>
+    <string name="rejected">Từ chối</string>
+    <string name="completed">Hoàn thành</string>
+    <string name="step_one_check_order">Bước 1: Kiểm tra đơn hàng</string>
+    <string name="step_two_close_order">Bước 2: Đóng đơn hàng</string>
+    <string name="continue_check">Tiếp tục</string>
+    <string name="shop_related">Cửa hàng liên quan</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.shop_related` appears to be unused
 Unused resources make applications larger and slow down builds.